hedera-node/hedera-file-service-impl/src/test/java/com/hedera/node/app/service/file/impl/test/handlers/FileAppendHandlerTest.java

```diff
@@ -19,9 +19,12 @@
 import static com.hedera.hapi.node.base.ResponseCodeEnum.FILE_DELETED;
 import static com.hedera.hapi.node.base.ResponseCodeEnum.INVALID_FILE_ID;
 import static com.hedera.node.app.spi.validation.ExpiryMeta.NA;
+import static org.assertj.core.api.AssertionsForClassTypes.assertThatThrownBy;
```

Comment:
the only change in that file are new imports?

Reply:
removed